src/migration/1733932616522-BookmarkUpdatedAt.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class BookmarkUpdatedAt1733932616522 implements MigrationInterface {
+    name = 'BookmarkUpdatedAt1733932616522'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "bookmark" ADD "updatedAt" TIMESTAMP NOT NULL DEFAULT now()`);
+        await queryRunner.query(`CREATE INDEX IF NOT EXISTS "IDX_bookmark_updatedAt" ON "bookmark" ("updatedAt") `);
```

Comment:
```suggestion
        await queryRunner.query(`CREATE INDEX IF NOT EXISTS "IDX_bookmark_updatedAt" ON "bookmark" ("updatedAt") DESC`);
```

because its used for ordering